src/model/APNSConfiguration.php

```diff
@@ -9,14 +9,29 @@ class APNSConfiguration {
 	public const APNS_ENVIRONMENT_SANDBOX = 'sandbox';
 	public const APNS_ENDPOINT_SANDBOX = 'https://api.development.push.apple.com/3/device/';
 
+	/** @var APNSCredentials */
 	private $credentials;
-	private $topic;
```

Comment:
I'm guessing this has been removed because Psalm noticed it was unused?

Reply:
Yep that's right!